src/Place/Place.php

```diff
@@ -147,7 +161,31 @@ public function updateMajorInfo(
         CalendarInterface $calendar,
         Theme $theme = null
     ) {
-        $this->apply(new MajorInfoUpdated($this->actorId, $title, $eventType, $address, $calendar, $theme));
+        $this->apply(
+            new MajorInfoUpdated(
+                $this->placeId,
+                $title,
+                $eventType,
+                $address,
+                $calendar,
+                $theme
+            )
+        );
+    }
+
+    /**
+     * @param Address $address
+     * @param Language $language
+     */
+    public function updateAddress(Address $address, Language $language)
```

Comment:
Ik heb dat momenteel niet voorzien omdat ik dan ook bij imports/updates van UDB2 de aggregate state zou moeten updaten. Anders kan het bv zijn dat je adres A ingeeft bij het aanmaken, er een update met adres B binnenkomt van UDB2, je wilt het terug aanpassen naar adres A maar dat gaat dan niet want de aggregate denkt nog steeds dat het huidige adres A is.

En de cdbxml parsen om de interne aggregate state te updaten zou redelijk wat extra werk zijn.

De voordelen van checken wat het adres was tegenover de nadelen van het parsen van de cdbxml zijn het dan niet echt waard volgens mij?